### PR TITLE
Doc/documentation

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: tue-documentation/setup [hunk 1]
```diff
@@ -10,15 +10,19 @@ function tue-make-documentation
 {
     if [ -n "$TUE_ROS_DISTRO" ] && [ -d "$TUE_SYSTEM_DIR" ]
     then
-        local build_tool
-        build_tool=$(cat "$TUE_SYSTEM_DIR"/devel/.built_by)
+        local build_tool=""
+        if [ -f "$TUE_SYSTEM_DIR"/devel/.built_by ]
```
Comment (LarsJanssenTUe) [inline tue-documentation/setup]: Why is this addition necessary?

File: tue-documentation/setup [hunk 2]
```diff
@@ -10,15 +10,19 @@ function tue-make-documentation
...
         case $build_tool in
         'catkin build')
             catkin document --workspace "$TUE_SYSTEM_DIR" "$@"
             ;;
         '')
-            catkin config --init --mkdirs --workspace "$TUE_SYSTEM_DIR" -DCMAKE_BUILD_TYPE=RelWithDebInfo "$@"
-            catkin document --workspace "$TUE_SYSTEM_DIR" "$@"
+            catkin config --init --mkdirs --workspace "$TUE_SYSTEM_DIR" --extend /opt/ros/"$TUE_ROS_DISTRO" -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCATKIN_ENABLE_TESTING=OFF
+            catkin build --workspace "$TUE_SYSTEM_DIR" "$@"
+            touch "$TUE_SYSTEM_DIR"/devel/.catkin # hack to allow overlaying to this ws while being empty
```
Comment (LarsJanssenTUe) [inline tue-documentation/setup]: Why was this hack not necessary before?

Comment (MatthijsBurgh) [reply tue-documentation/setup]: Explicit workspace extending.

File: tue-documentation/setup (hunk 1 above)
Comment (MatthijsBurgh) [reply tue-documentation/setup]: So you don't get the error if the file doesn't exists. This is the case on a new pc or after cleaning your ws.
